source/isaaclab/isaaclab/sensors/ray_caster/ray_caster.py

```diff
@@ -157,6 +168,10 @@ def _initialize_warp_meshes(self):
 
         # read prims to ray-cast
         for mesh_prim_path in self.cfg.mesh_prim_paths:
+            # check if mesh already casted into warp mesh
```

Comment:
Shouldn't

```python
self.meshes[mesh_prim_path] = wp_mesh
```

Be:

```python
RayCaster.meshes[mesh_prim_path] = wp_mesh
```

I thought all write operations need to happen through class for classvar